llvm/include/llvm/CodeGen/SelectionDAG.h

```diff
@@ -1595,10 +1595,10 @@ class SelectionDAG {
   /// the target's desired shift amount type.
   SDValue getShiftAmountOperand(EVT LHSTy, SDValue Op);
 
-  /// Create the DAG equivalent of vector_partial_reduce where Op1 and Op2 are
-  /// its operands and ReducedTY is the intrinsic's return type.
-  SDValue getPartialReduceAdd(SDLoc DL, EVT ReducedTy, SDValue Op1,
-                              SDValue Op2);
+  /// Expands partial reduce node which can't be lowered to wide add or dot
+  /// product instruction(s)
```

Comment:
SelectionDAG.h is not target specific, so I don't think this should mention the wide add/dot product lowering (which is only for AArch64). Also, could you document the operands (ReducedTY, Op1, Op2)?